Raspi/Test/mock_wireless_module.py

```diff
@@ -0,0 +1,159 @@
+import time
+import argparse
+import json
+import paho.mqtt.client as mqtt
+
+# my library
+from MockSensor import MockSensor
+
+parser = argparse.ArgumentParser(
+    description='MQTT wireless sensor test script that sends fake data',
+    add_help=True)
+parser.add_argument(
+    '-t', '--time', action='store', type=int,
+    default=float('Inf'), help="""Length of time to send data in seconds
+    (duration). If nothing is selected it will continuously send out data.""")
+parser.add_argument(
+    '-r', '--rate', action='store', type=float, default=1,
+    help="""Rate of data in data sent per second. Default is 1 data pack sent
+    per second.""")
+parser.add_argument(
+    '--host', action='store', type=str, default="192.168.100.100",
+    help="""Address of the MQTT broker. If nothing is selected it will
+    default to 192.168.100.100.""")
+parser.add_argument(
+    '--id', action='store', type=int, default=None,
+    help="""Specify the sensor to produce fake data. eg. --id 1 specifies that
+    sensor 1 only produces data. If nothing is given all sensors will be
+    active.""")
+
+# Generate the fake sensors with average values
+s_steering_angle = MockSensor(10)
+s_c02 = MockSensor(325)
+s_temperature = MockSensor(25)
+s_humidity = MockSensor(85)
+s_reed_velocity = MockSensor(50)
+s_battery = MockSensor(80)
+s_accelerometer = MockSensor(("x", 90),
+                             ("y", 90),
+                             ("z", 90))
+s_gyroscope = MockSensor(("x", 90),
+                         ("y", 90),
+                         ("z", 90))
+s_GPS = MockSensor(("speed", 50),
+                   ("satellites", 10),
+                   ("latitude", 25),
+                   ("longitude", 25),
+                   ("altitude", 50),
+                   ("course", 0))
+
+
+def send_fake_data(client, duration, rate, sensor_ID):
+    """ Send artificial data over MQTT for each sensor chanel. Sends [rate] per
+    second for [duration] seconds"""
+
+    start_time = round(time.time(), 2)
+    total_time = 0
+
+    while total_time < duration:
+        current_time = round(time.time(), 2)
+        total_time = round(current_time - start_time, 2)
+        print()     # Newline for clarity
+
+        # Wireless Sensor 1 (Middle)
+        if sensor_ID == 1 or sensor_ID is None:
+            sensor_data = {
```

Comment:
Another TODO would be to create another function that actually outputs the wireless module data output. This will make it a lot easier to write tests! Also happy for you to fix this in another PR!